<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratedModel</class>
 <widget class="QDialog" name="GeneratedModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MingLiU</family>
    </font>
   </property>
   <property name="text">
    <string>Generated Model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MingLiU</family>
    </font>
   </property>
   <property name="text">
    <string>(Researcher)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="duration_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Save Model</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
